spec/big_files/big_files_spec.rb

```diff
@@ -71,10 +78,32 @@ def expect_source_globber_used(glob, exclude_glob)
 
         it 'runs' do
           expect_source_globber_used(glob, exclude_glob)
-          file_1 = expect_file_processed('file_1', 1)
-          file_2 = expect_file_processed('file_2', 2)
+          file_1 = expect_file_processed('file_1', 4)
+          file_2 = expect_file_processed('file_2', 3)
+          file_3 = expect_file_processed('file_3', 2)
+          file_4 = expect_file_processed('file_4', 1)
+
           allow(file_1).to receive(:<=>).with(file_2).and_return(1)
+          allow(file_1).to receive(:<=>).with(file_3).and_return(1)
+          allow(file_1).to receive(:<=>).with(file_4).and_return(1)
+
           allow(file_2).to receive(:<=>).with(file_1).and_return(-1)
+          allow(file_2).to receive(:<=>).with(file_3).and_return(1)
+          allow(file_2).to receive(:<=>).with(file_4).and_return(1)
+
+          allow(file_3).to receive(:<=>).with(file_1).and_return(-1)
+          allow(file_3).to receive(:<=>).with(file_2).and_return(-1)
+          allow(file_3).to receive(:<=>).with(file_4).and_return(1)
+
+          allow(file_4).to receive(:<=>).with(file_1).and_return(-1)
+          allow(file_4).to receive(:<=>).with(file_2).and_return(-1)
+          allow(file_4).to receive(:<=>).with(file_3).and_return(-1)
+
+          expect_file_output('file_1', 4)
+          expect_file_output('file_2', 3)
+          expect_file_output('file_3', 2)
+          expect_file_output('file_4', 1) if num_files && num_files.to_i >= 4
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.